proteus-metrics-prometheus/src/main/java/io/netifi/proteus/prometheus/ProteusPrometheusBridge.java

```diff
@@ -90,7 +90,7 @@ public static void main(String... args) {
     logger.info("access key - {}", accessKey);
 
     Proteus proteus =
-        Proteus.builder()
+        Proteus.tcp()
```

Comment:
Is the builder() method left on the api for backwards compatibility?

Reply:
Yup. Marked as deprecated -> https://github.com/netifi-proteus/proteus-java/pull/41/files#diff-031fa89a1a13d01ef085d6a5af1e48b1R114